net/nebula/Makefile

```diff
@@ -0,0 +1,77 @@
+# Copyright 2021 Stan Grishin (stangri@melmac.net)
+# This is free software, licensed under the MIT License.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nebula
+PKG_VERSION:=1.3.0
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Stan Grishin <stangri@melmac.net>
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/slackhq/nebula/archive/
+PKG_HASH:=b94fba0251a4a436e25b127d0b9bc0181b991631f1dc8e344b1c8e895b55375d
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+GO_PKG:=github.com/slackhq/nebula
+GO_PKG_LDFLAGS_X:=main.Build=$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/nebula/default
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=nebula
+  URL:=https://github.com/slackhq/nebula
+  DEPENDS:=$(GO_ARCH_DEPENDS)
+endef
+
+define Package/nebula
+$(call Package/nebula/default)
+  DEPENDS+= +kmod-tun
+endef
+
+define Package/nebula-cert
+$(call Package/nebula/default)
+  VARIANT:=cert
```

Comment:
Is `VARIANT` here still necessary?

Reply:
Is it better to define two different packages?

Reply:
They already are two different packages. `VARIANT` is usually used when compilation is different for each variant, e.g. [openvpn](https://github.com/openwrt/packages/blob/6b9fdf2e77cdccb74902b84938d1744d8b8aafea/net/openvpn/Makefile#L53-L58), [less](https://github.com/openwrt/packages/blob/6b9fdf2e77cdccb74902b84938d1744d8b8aafea/utils/less/Makefile#L62-L70). AFAICT nebula and nebula-cert are packaging different binaries from the same compilation process.

Reply:
> They already are two different packages. `VARIANT` is usually used when compilation is different for each variant, e.g. [openvpn](https://github.com/openwrt/packages/blob/6b9fdf2e77cdccb74902b84938d1744d8b8aafea/net/openvpn/Makefile#L53-L58), [less](https://github.com/openwrt/packages/blob/6b9fdf2e77cdccb74902b84938d1744d8b8aafea/utils/less/Makefile#L62-L70). AFAICT nebula and nebula-cert are packaging different binaries from the same compilation process.

Yeah, makes sense, was the leftover from nebula/nebula-full I guess. I pushed an update without variants.